reactioncommerce/reaction
# refactor: create verifySMTPEmailSettings GraphQL mutation, replace meteor version

Resolves parts of #5529
Impact: minor
Type: feat|refactor

## Issue
`email/verifySettings` was a meteor method which used `getGraphQLContextInMeteorMethod`

## Solution
Rewrite the meteor method to be a `verifySMTPEmailSettings` GraphQL mutation, which in turn will remove usage of `getGraphQLContextInMeteorMethod`.

The color of the status indicator is not reactive. This is not a regression, it is currently not reactive in `master`. Since there are other Meteor methods and more refactors of this code to come in the future, I've chosen to leave this as is for now.

## Breaking changes
Any custom code which used the `email/verifySettings` meteor method will need to be updated to use the GraphQL mutation

## Testing
1. Add correct email settings and see they come back as verified
1. Add incorrect email settings and see they come back as an error
